server/controllers/board.js

```diff
@@ -0,0 +1,111 @@
+const Board = require("../models/Board");
+const Card = require("../models/Card");
+const asyncHandler = require("express-async-handler");
+const BoardColumn = require("../models/BoardColumn");
+
+// @route POST /users
+// @desc Search for users
+// @access Private
+exports.listBoard = asyncHandler(async (req, res, next) => {
+  
+  const board = await Board.find({
+    user: req.user.id
+  }).populate({
+    path: 'columns',
+    populate: {
+      path: 'cards'
+    }
+  });
+  
+
+  res.status(200).json({ board });
```

Comment:
I see that u haven't done error handling for any , can you add that too
 E.g
if (!board) {
    res.status(404);
    throw new Error("No board found with given id");
  }

Reply:
Sure, I will create the error handling.